src/sed/plugins/locator/nll/CMakeLists.txt

```diff
@@ -27,20 +45,34 @@ SET( ${PACKAGE_NAME}_SOURCES
 	${LIBNLL_ALOMAX_OCTTREE_SOURCES}
 	${LIBNLL_ALOMAX_RAN1_SOURCES}
 	${LIBNLL_ALOMAX_VECTOR_SOURCES}
+    ${LIBNLL_OPEN_MEMSTREAM_OSX_SOURCES}
 	nll_locator.cpp
 )
+ENDIF(MACOSX)
```

Comment:
I would prefer to just add the MACOSX specific files, e.g.:

```cmake
IF(MACOSX)
  SET(${PACKAGE_NAME}_SOURCES
    ${PACKAGE_NAME}_SOURCES
    ${LIBNLL_OPEN_MEMSTREAM_OSX_SOURCES}
  )
ENDIF(MACOSX)
```

Then we don't need an additional Linux block.